### ISSUE TITLE
Deprecated: trim(

### ISSUE THREAD

**Issue by yovannygomez** [BrainMaestro/composer-git-hooks#138]

**Comment by ZejdCicak** [comment]
I have fixed this and it has been merged to master but it was not tagged yet, I have no idea why.

**Comment by leymannx** [comment]
@BrainMaestro – Can we get this tagged please. 🙏🏻 🙂

**Comment by yovannygomez** [comment]
@ZejdCicak you're right. I already saw your merged. The commit id was 3d197c4. But I don't know how we can generate a new tag.
